### PR TITLE
CORE-19128: Support multiple CAs in the gateway server

### PR DESCRIPTION
**Pull request by yift-r3**
Add support for more than one certificate authority in the gateway.

### CODE REVIEW

**Review by dickon — DISMISSED** [review]
LGTM

File: components/gateway/src/integrationTest/kotlin/net/corda/p2p/gateway/GatewayIntegrationTest.kt [hunk 1]
```diff
@@ -1525,6 +1526,74 @@ internal class GatewayIntegrationTest : TestBase() {
                 }
             }
         }
```
Comment (nikinagy) [inline components/gateway/src/integrationTest/kotlin/net/corda/p2p/gateway/GatewayIntegrationTest.kt:1528]: nit: can we add new empty line here?

**Review by nikinagy — DISMISSED** [review]
LGTM, just added a nit

**Review by nikinagy — APPROVED** [review]
LGTM

**Review by dickon — APPROVED** [review]
LGTM after new push

**Pull request merged**